README.md

```diff
@@ -1,7 +1,7 @@
 **Interested in contributing a code example?** 
 
 Please take a look at our [contribution guidelines](CONTRIBUTING.md) before
-getting started. Thanks!
+getting started. Thank you!
```

Comment:
Does the CONTRIBUTING need to be updated for Dash examples? 
```
Tests that are discoverable through python -m unittest discover
```
Dash uses pytest